consensus/src/counters.rs

```diff
@@ -2,74 +2,64 @@
+pub static ref CURRENT_ROUND: IntGauge = register_int_gauge!("libra_consensus_current_round", "This counter is set to the last round reported by the local pacemaker.").unwrap();
 
 /// Count of the committed blocks since last restart.
-pub static ref COMMITTED_BLOCKS_COUNT: IntCounter = OP_COUNTERS.counter("committed_blocks_count");
+pub static ref COMMITTED_BLOCKS_COUNT: IntCounter = register_int_counter!("libra_consensus_committed_blocks_count", "Count of the committed blocks since last restart.").unwrap();
 
 /// Count of the committed transactions since last restart.
-pub static ref COMMITTED_TXNS_COUNT: IntCounter = OP_COUNTERS.counter("committed_txns_count");
+pub static ref COMMITTED_TXNS_COUNT: IntCounterVec = register_int_counter_vec!("libra_consensus_committed_txns_count", "Count of the transactions since last restart. state is success or failed", &["state"]).unwrap();
 
-/// Count of success txns in the blocks committed by this validator since last restart.
-pub static ref SUCCESS_TXNS_COUNT: IntCounter = OP_COUNTERS.counter("success_txns_count");
+/// Histogram of idle time of spent in event processing loop
+pub static ref EVENT_PROCESSING_LOOP_IDLE_DURATION_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_event_processing_loop_idle_duration_s", "Histogram of idle time of spent in event processing loop").unwrap());
 
-/// Count of failed txns in the committed blocks since last restart.
-/// FAILED_TXNS_COUNT + SUCCESS_TXN_COUNT == COMMITTED_TXNS_COUNT
-pub static ref FAILED_TXNS_COUNT: IntCounter = OP_COUNTERS.counter("failed_txns_count");
-
-/// Histogram of idle time (ms) of spent in event processing loop
-pub static ref EVENT_PROCESSING_LOOP_IDLE_DURATION_S: DurationHistogram = OP_COUNTERS.duration_histogram("event_processing_loop_idle_duration_s");
-/// Histogram of busy time (ms) of spent in event processing loop
-pub static ref EVENT_PROCESSING_LOOP_BUSY_DURATION_S: DurationHistogram = OP_COUNTERS.duration_histogram("event_processing_loop_busy_duration_s");
+/// Histogram of busy time of spent in event processing loop
+pub static ref EVENT_PROCESSING_LOOP_BUSY_DURATION_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_event_processing_loop_busy_duration_s", "Histogram of busy time of spent in event processing loop").unwrap());
 
 /// Counters(queued,dequeued,dropped) related to proposals channel
-pub static ref PROPOSAL_DROPPED_MSGS: IntCounter = OP_COUNTERS.counter("proposal_dropped_msgs_count");
-pub static ref PROPOSAL_ENQUEUED_MSGS: IntCounter = OP_COUNTERS.counter("proposal_enqueued_msgs_count");
-pub static ref PROPOSAL_DEQUEUED_MSGS: IntCounter = OP_COUNTERS.counter("proposal_dequeued_msgs_count");
+pub static ref PROPOSAL_DROPPED_MSGS: IntCounter = register_int_counter!("libra_consensus_proposal_dropped_msgs_count", "Counters(queued,dequeued,dropped) related to proposals channel").unwrap();
```

Comment:
Same as those, they can be one counter `libra_consensus_proposal_msgs_count`, with state=dropped, enqueued, dequeued

Reply:
Updated